include/ensmallen_bits/pso/pso.hpp

```diff
@@ -159,16 +159,16 @@ class PSOType
   size_t& NumParticles() { return numParticles; }
 
   //! Retrieve value of lowerBound.
-  size_t LowerBound() const { return lowerBound; }
+  const arma::vec LowerBound() const { return lowerBound; }
```

Comment:
Did you forget to add a reference `&`?
```c++
const arma::vec&
```
I thought the copy-constructor of `arma::vec` isn't trivial.

Reply:
You are absolutely right, thanks again for pointing this out.